spec/models/conference_spec.rb

```diff
@@ -1639,4 +1639,27 @@
       expect(free_ticket.price_cents).to eq(0)
     end
   end
+
+  describe 'after_update' do
+    let(:conference) { create(:full_conference) }
```

Comment:
No specific reason just prefer to use full_conference with all basic things set. But yes conference will do just fine here. :+1: